src/Util/ClassSourceManipulator.php

```diff
@@ -58,6 +58,8 @@ final class ClassSourceManipulator
     public function __construct(string $sourceCode, bool $overwrite = false, bool $useAnnotations = true, bool $fluentMutators = true)
     {
         $this->overwrite = $overwrite;
+        // is this needed? https://github.com/symfony/maker-bundle/pull/920 we discussed about moving "useAttributes" into the methods itselfs
+        // which would make "useAnnotations" obsolete
```

Comment:
This property really means:

> Should Doctrine metadata (whether that is annotations or attributes) be added when `addProperty()` is called?

So, it's almost a "mode" that we want to activate on this entire manipulator. I think it's fine as a property, but it could be renamed to:

> shouldAddDoctrineMetadatatoNewProperties

A mouthful, but much more precise.